verksamhetsOmraden/verksamhetsOmraden.html

```diff
@@ -3,35 +3,39 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
     <title>Document</title>
+
     
 </head>
 <body>
-    <h2>
-    Konst- och kulturutbildningar
-    </h2>
-
-    <p>
-        Inom konst- och kulturutbildningar finns det flera olika inriktningar 
+    
+    <div class="grid">
+ 
+    <h2 style="color:indigo;"> Myndighetens uppgifter för konst-och kulturutbildningar
+    
+    
+    <p style="color: grey;"> Inom konst- och kulturutbildningar finns det flera
+        olika inriktningar
         som bildkonst, musik, teater, dans, mode,
         skrivande, arkitektur, silversmide och keramik för att nämna några.
-    </p>
-    <h2>
-        Myndighetens uppgifter för konst- och kulturutbildningar
-    </h2>
-    <ul>
-        <li> Fatta beslut om vilka utbildningsanordnare som ska beviljas statligt stöd (statsbidrag och/eller
-        studiestödsberättigande) </li>
+    
+    
+    <ul style="color: grey;">
```

Comment:
lets try not putting css in the html file, it gets messy fast

Reply:
Removed